contracts/contracts/RequestManager.sol

```diff
@@ -119,19 +125,27 @@ contract RequestManager is Ownable, LpWhitelist, RestrictedCalls, Pausable {
         address stakeRecipient
     );
 
-    event FinalityPeriodUpdated(uint256 targetChainId, uint256 finalityPeriod);
-
     /// Emitted when token object of a token address is updated.
     ///
     /// .. seealso:: :sol:func:`updateToken`
     event TokenUpdated(
         address tokenAddress,
         uint256 transferLimit,
-        uint96 minLpFee,
+        uint256 rate,
```

Comment:
Should we also consider renaming this field, in light of the other "ETH/USD" discussion?

Reply:
yes, I would rename all occurences of rate throughout the code base